language-agent/Meter.proto

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "org.apache.skywalking.apm.network.language.agent.v3";
+
+import "common/Common.proto";
+
+service MeterReportService {
+    rpc collect (stream MeterData) returns (Commands) {
+    }
+}
+
+// Label in the meter
+message Label {
+    string name = 1;
+    string value = 2;
+}
+
+// Work for histogram, defined the bucket
+message MeterBucketValue {
+    int32 bucket = 1;
```

Comment:
```suggestion
    // The bucket is [0, $value]
    int32 bucket = 1;
```

Is this explanation right? Are we using the Prometheus style too? I remember we said we have min/max values boundaries. Do I miss anything?

Reply:
We will have boundaries settings in the `API` level, `exceptMinValue` and `exceptMinValue` will work on it. But at the default, it also is `Integer.MIN_VALUE` and `Integer.MAX_VALUE`. 
Using Prometheus style to relate the bucket and value. such as [previous bucket, $value].

Reply:
I think the boundary should be very clear at the protocol level. MIN and MAX seem not a good way. API is another thing, let's talk later.

> Using Prometheus style to relate the bucket and value. such as [previous bucket, $value].

I am not asking to use it or not. I am asking about what is your design?

Reply:
I updated the comments based on our offline discussion. Take a look.

Reply:
Thanks for helping me to fix the document.